datasets/klue/klue.py

```diff
@@ -0,0 +1,519 @@
+                            index = []
+                            word_form = []
+                            lemma = []
+                            pos = []
+                            head = []
+                            deprel = []
+                            sentence = row[1]
+                        else:
+                            index.append(row[0])
+                            word_form.append(row[1])
+                            lemma.append(row[2])
+                            pos.append(row[3])
+                            head.append(row[4])
+                            deprel.append(row[5])
+                    else:  # new line
+                        assert len(index) == len(word_form) == len(lemma) == len(pos) == len(head) == len(deprel)
+                        yield id_, {
+                            "sentence": sentence,
+                            "index": index,
+                            "word_form": word_form,
+                            "lemma": lemma,
+                            "pos": pos,
+                            "head": head,
+                            "deprel": deprel,
+                        }
+
+        if self.config.name == "mrc":
+            with open(data_file, encoding="UTF-8") as f:
+                f = json.load(f)
+                for example in f["data"]:
+                    title = example.get("title", "")
+                    news_category = example.get("news_category", "")
+                    source = example["source"]
+                    for paragraph in example["paragraphs"]:
+                        context = paragraph["context"].strip()
+                        for qa in paragraph["qas"]:
+                            guid = qa["guid"]
+                            question_type = qa["question_type"]
+                            is_impossible = qa["is_impossible"]
+                            question = qa["question"].strip()
+
+                            if "plausible_answers" in qa:
+                                qa["answers"].extend(qa["plausible_answers"])
+                            answer_starts = [answer["answer_start"] for answer in qa["answers"]]
+                            answers = [answer["text"].strip() for answer in qa["answers"]]
+
+                            yield guid, {
+                                "title": title,
+                                "context": context,
+                                "news_category": news_category,
+                                "source": source,
+                                "guid": guid,
+                                "question_type": question_type,
+                                "is_impossible": is_impossible,
+                                "question": question,
+                                "answers": {
+                                    "answer_start": answer_starts,
+                                    "text": answers,
+                                },
+                            }
```

Comment:
I suggest to use `id_` instead of `guid` for consistency with other datasets.

```suggestion
        if self.config.name == "mrc":
            with open(data_file, encoding="UTF-8") as f:
                f = json.load(f)
                id_ = -1
                for example in f["data"]:
                    title = example.get("title", "")
                    news_category = example.get("news_category", "")
                    source = example["source"]
                    for paragraph in example["paragraphs"]:
                        context = paragraph["context"].strip()
                        for qa in paragraph["qas"]:
                            guid = qa["guid"]
                            question_type = qa["question_type"]
                            is_impossible = qa["is_impossible"]
                            question = qa["question"].strip()

                            if "plausible_answers" in qa:
                                qa["answers"].extend(qa["plausible_answers"])
                            answer_starts = [answer["answer_start"] for answer in qa["answers"]]
                            answers = [answer["text"].strip() for answer in qa["answers"]]
                            id_ += 1

                            yield id_, {
                                "title": title,
                                "context": context,
                                "news_category": news_category,
                                "source": source,
                                "guid": guid,
                                "question_type": question_type,
                                "is_impossible": is_impossible,
                                "question": question,
                                "answers": {
                                    "answer_start": answer_starts,
                                    "text": answers,
                                },
                            }
```